resources/backup_scripts/code-backup.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
```

Comment:
Also, I wanted to check if you ran this already. I see files in the `warmshowers-files-backup` directory, so I assume so.